src/login/logind-button.c

```diff
@@ -19,6 +19,8 @@
 
 #define ULONG_BITS (sizeof(unsigned long)*8)
 
+#define LONG_PRESS_DURATION (5 * USEC_PER_SEC)
```

Comment:
we usually suffix time constants with `_USEC`. i.e maybe name LONG_PRESS_DURATION_USEC or LONG_PRESS_USEC or so

Reply:
you changed this from 4 to 5 here btw, by accident?

Reply:
I would prefer having 5s since this is not user-adjustable.
5s is a sort of an industry standard and all of the boards from routers, AP-s, switches, etc used at least 5s seconds to trigger
the action on button hold instead of just the click one like the reboot.